### PR TITLE
Integrate with travis CI

### PR DESCRIPTION
**Pull request by kumaya**
- include minimalistic .travis.yml
  - remove unused dependencies
  - modify .gitignore file
  - modify `make test` command

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: go
+sudo: required
+
+services:
+  - docker
+go:
+  - 1.9
+  - "1.10"
```
Comment (jkraj) [inline .travis.yml]: nit: why double quotes for this.

File: Makefile [hunk 2]
```diff
@@ -1,36 +1,38 @@
-VERSION := $(shell git describe --dirty=-dirty --always)
-
-APP_NAME := atlas-db
-
 # Absolute github repository name.
```
Comment (jkraj) [inline Makefile]: Can you update this file based on my #21

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+script:
+  - make test
```
Comment (jkraj) [inline .travis.yml]: can we add _make fmt_ as well.

File: .travis.yml (hunk 1 above)
Comment (kumaya) [reply .travis.yml]: without quotes travis CI interprets go version "1.10" as "1.1"

File: .travis.yml (hunk 3 above)
Comment (kumaya) [reply .travis.yml]: `make test` includes `fmt` test as well. It is part of Makefile.

**Comment by kumaya** [comment]
Will rebase and push updated changes once #21 is merged

**Comment by kumaya** [comment]
huge overhead to rebase. Closing in favour of new commit

**Pull request closed without merging**